cmd/create/operatorroles/by_prefix.go

```diff
@@ -195,6 +195,10 @@ func handleOperatorRoleCreationByPrefix(r *rosa.Runtime, env string,
 			r.Reporter.Infof(fmt.Sprintf("To create a cluster with these roles, run the following command:\n"+
 				"\trosa create cluster --sts --oidc-config-id %s --operator-roles-prefix %s%s",
 				args.oidcConfigId, args.prefix, hostedCpOutputParam))
+			if args.installerRoleArn != "" {
+				r.Reporter.Infof("Please consider using the same installer role '%s', when creating the cluster.",
```

Comment:
@oriAdler Is it an error if the user doesn't use the same installer role? Reading through OCM-4924, it looks like it the cluster creation won't work. Should it be clearer e.g 

```
When using these operator roles for a cluster, you must use installer role '%s' to create the cluster e.g rosa create cluster --role-arn '%s'
```

Reply:
Improved the info message, thanks.
I didn't add the flag, as it makes the `create cluster` command less user-friendly:
`If we provide only the installer role ARN flag in create cluster using --role-arn (without other account role flags), it makes the user to input manually support and worker role instead of the CLI detecting the roles in the interactive mode. `
WDYT?

Reply:
@oriAdler  What's the easiest way for us to tell the user how to use the specific installer role? My reaction was "OK, I need to use a specific installer role...how do I do that?"

Is it the default if the user doesn't do anything e.g the CLI will detect the correct installer role?

cheers,

Rob

Reply:
Right, if that is the case we need to provide the `--role-arn` flag.

Reply:
Thanks @oriAdler . I think that's really clear now.

cheers,

Rob